src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -38,7 +30,7 @@ enum
 };
 
 static vaddr_t
-boot_deps_map_sect(int spdid, vaddr_t dest_daddr)
+boot_deps_map_sect(spdid_t spdid, vaddr_t dest_daddr)
```

Comment:
All of these should really now be `compid_t`, but that is a huge repo-wide update, so, for now, these `spdid_t` changes are great.